sunpy/time/tests/test_time.py

```diff
@@ -288,9 +288,9 @@ def test_parse_time_astropy_formats(ts, fmt):
 def test_parse_time_int_float():
```

Comment:
It seems this is failing on the oldest deps test with an older version of astropy where it raises a different message.

I am not sure how best to handle that fact. We could xfail it for old astropy versions or we can change the message based on astropy versions.

Reply:
I was concerned with this test failing. Thank you for pointing this out. At this moment, I think we can XFAIL this. We can take this as a separate minor issue. In that issue we can propose to add conditionals to handle the older astropy version.

Reply:
I don't see the need to open and track an issue to fix this, we should fix it here.

Reply:
Sure, I think then we should XFAIL this since this runs fine on the latest versions and doesn't seem to break any core-functionality. Although proceeding with this option can make it harder to detect unexpected changes since we are not handling the older versions separately.

Reply:
Sure, can you xfail based on the version of astropy?

Reply:
I can do that. Should I XFAIL below the current version of astropy (6.1.4)?

Reply:
Good question, unless we work out the correct version, then I would just say yeah, do less than <6.1 and see if that works.

Reply:
I have implemented with "6.1".